storage/gc_queue.go

```diff
@@ -219,12 +219,9 @@ func (gcq *gcQueue) process(now proto.Timestamp, rng *Replica) error {
 	switch len(gcArgs.Keys) {
 	case 0:
 		return nil
-	case 1:
-		gcArgs.Key = gcArgs.Keys[0].Key
-		gcArgs.EndKey = gcArgs.Key.Next()
 	default:
 		gcArgs.Key = gcArgs.Keys[0].Key
-		gcArgs.EndKey = gcArgs.Keys[len(gcArgs.Keys)-1].Key
+		gcArgs.EndKey = gcArgs.Keys[len(gcArgs.Keys)-1].Key.Next()
```

Comment:
ok, the fix here seems to make sense (otherwise you'd always exclude the last key).